### DISCUSSION TITLE
testrepo/qna

### DISCUSSION THREAD

**Discussion by giscus** [fineapple1818/testrepo#3] [Q&A]
# testrepo/qna

A starter template for a Jeykll site using the Just the Docs theme!

https://fineapple1818.github.io/testrepo/qna.html

**Comment by fineapple1818** [comment]
질의응답 테스트

**Comment by fineapple1818** [comment]
질의응답 테스트

**Comment by fineapple1818** [comment]
반가워요/-?

**Comment by seooyxx** [comment]
너 누군데?
